transcribed/page0060.txt

```diff
@@ -45,8 +45,8 @@
 045     001 017         ANDM T,1(P)
 046     001 050         CONO 430000+APRCHN
 047     001 039         MOVEI T,BADBTS
-048     118 036         OIR T,MSKST(U)
+048     118 036         IOR T,MSKST(U)
 049     001 017         AND T,1(P)
-050     006 007         JUMPE T,CLKB5
```

Comment:
I think it's 007, not 017.

Reply:
You were right, thanks for fixing my pull requests. :)